jax/_src/lib/xla_bridge.py

```diff
@@ -350,7 +350,8 @@ def backends():
           # we expect a RuntimeError.
           err_msg = f"Unable to initialize backend '{platform}': {err}"
           if config.jax_platforms:
-            raise RuntimeError(err_msg)
+            setup_msg = f" (set JAX_PLATFORMS='' to choose backend automatically)"
```

Comment:
One more thought: how about `to automatically choose an available backend)`

Also you don't need the `f` since you're not formatting anything. Also you can use `err_msg +=` if you want

Reply:
Sounds good.